packages/gatsby/cache-dir/rspack-init.js

```diff
@@ -0,0 +1 @@
+globalThis.__resourceQuery = `?hot=true`
```

Comment:
without this `@rspack/dev-client` was failing to compile on https://github.com/web-infra-dev/rspack/blob/b9779464b7c5f6ad067ecc407e8548b9b31179dd/packages/rspack-dev-client/src/index.ts#L46